httpclient/httpclient.go

```diff
@@ -327,6 +329,15 @@ func Propagation(propagation bool) func(*Request) {
 	}
 }
 
+// FollowRedirects allows the body to be copied to a new request to the Location
+// returned by a redirect response, or to fully retry an HTTP/2 GOAWAY frame with no
+// associated stream error.
```

Comment:
citation needed - the [returned error in honeycomb](https://ui.honeycomb.io/circleci/datasets/execution/result/k5tCiX9eQa4/trace/oG53MWtJALn?fields[]=c_name&fields[]=c_derived.service&span=5ed205cedc5669a7) seems to indicate the right thing will happen if we implement `req.GetBody`.